### PR TITLE
[Bugfix] Change benchmark test repo to our own example repo

### PR DESCRIPTION
**Pull request by bodhinandach**
**Describe the PR**
This PR modify the CI benchmark testing to our own example repo.

**Related Issues/PRs**
This fix Issue https://github.com/geomechanics/mpm/issues/41

**Additional context**
N/A

### CODE REVIEW

**Review by jgiven100 — APPROVED** [review]
:cowboy_hat_face:

File: .circleci/config.yml [hunk 1]
```diff
@@ -107,32 +107,32 @@ jobs:
...
+            python3 ./benchmarks/explicit_single_phase/validation/sliding_block_inclined_boundary/test_benchmark.py
       # Stress
       - run:
           name: Uniaxial stress
           command: |
             cd /home/cbgeo/project/build
-            ./mpm -f ./benchmarks/2d/uniaxial_stress/
-            python3 ./benchmarks/2d/uniaxial_stress/test_benchmark.py
+            ./mpm -f ./benchmarks/explicit_single_phase/validation/uniaxial_stress/
+            python3 ./benchmarks/explicit_single_phase/validation/uniaxial_stress/test_benchmark.py
```
Comment (jgiven100) [inline .circleci/config.yml:126]: Wait, not actually approving this. All of the input files should be a subdirectory called "inputs"

For example:
```
python3 ./benchmarks/explicit_single_phase/validation/uniaxial_stress/inputs/test_benchmark.py
```

Comment (jgiven100) [reply .circleci/config.yml:126]: PR opened https://github.com/geomechanics/mpm-examples/pull/5

Comment (bodhinandach) [reply .circleci/config.yml]: OK! Does it look ok now?

**Review by jgiven100 — APPROVED** [review]

**Pull request merged**